src/main/java/com/moodysalem/TimezoneMapper.java

```diff
@@ -10,16 +9,7 @@
     /**
      * Prevent instantiating an instance of this class
      */
-    private TimezoneMapper() {
-    }
-
-    public static ZoneId tzAt(double lat, double lng) {
```

Comment:
I agree, it is a useful method. However, it's using the Java 8 time api with ZoneId, and it's incompatible with Java 7.